### PR TITLE
Merged the two modal confirmation into one

### PR DESCRIPTION
**Pull request by 0tuedon**
Screenshot 2023-07-08 at 18 48 59
new look of the modal @adamjonas  please confirm if this design is good. #101

### CODE REVIEW

File: src/components/alerts/SubmitTranscriptAlert.tsx [hunk 1]
```diff
@@ -127,15 +115,7 @@ const PromptStepOne = forwardRef<HTMLButtonElement, PromptOneProps>(
...
-            <Button
-              display="block"
-              size="sm"
-              mx="auto"
-              colorScheme="blue"
-              onClick={() => setPromptStep(2)}
-            >
-              Ready to ship! <span style={{}}>🚢</span>
-            </Button>
+            {/* Pr */}
```
Comment (Extheoisah) [inline src/components/alerts/SubmitTranscriptAlert.tsx:118]: remove comments

File: src/components/alerts/SubmitTranscriptAlert.tsx [hunk 2]
```diff
@@ -23,9 +23,7 @@ type Props = {
   prRepo: TranscriptSubmitOptions;
 };
 
-type PromptTwoProps = {
-  setPromptStep: Dispatch<SetStateAction<number>>;
-} & Omit<Props, "isOpen">;
+type PromptTwoProps = {} & Omit<Props, "isOpen">;
```
Comment (Extheoisah) [inline src/components/alerts/SubmitTranscriptAlert.tsx:26]: why do have an object as an extended type?

File: src/components/alerts/SubmitTranscriptAlert.tsx [hunk 3]
```diff
@@ -146,30 +126,19 @@ const PromptStepOne = forwardRef<HTMLButtonElement, PromptOneProps>(
...
-  ({ onCancel, prRepo, onSubmit }, ref) => {
+  ({ onCancel, prRepo, onSubmit }) => {
     return (
       <>
-        <AlertDialogHeader fontSize="lg" fontWeight="bold">
-          Let&apos;s review your edits
-        </AlertDialogHeader>
         <AlertDialogBody>
-          Are you sure you want to submit your review? This would create a PR on{" "}
+          {/* Are you sure you want to submit your review? This would create a PR on{" "} */}
```
Comment (Extheoisah) [inline src/components/alerts/SubmitTranscriptAlert.tsx:133]: If we're settling for the new text, we can remove this.

**Review by Extheoisah — CHANGES_REQUESTED** [review]
Looks good. It'd be nice if you could have a clean commit history going forward. Generally, we want to avoid merge commits and do a rebase instead.
I left some comments. Otherwise, @adamjonas can confirm he's satisfied with the modal

File: src/components/alerts/SubmitTranscriptAlert.tsx (hunk 3 above)
Comment (0tuedon) [reply src/components/alerts/SubmitTranscriptAlert.tsx:133]: okay, wanted to confirm if this is what we want first

**Comment by 0tuedon** [comment]
> Looks good. It'd be nice if you could have a clean commit history going forward. Generally, we want to avoid merge commits and do a rebase instead. I left some comments. Otherwise, @adamjonas can confirm he's satisfied with the modal

it's still says a merge confilcts?

**Comment by Extheoisah** [comment]
> > Looks good. It'd be nice if you could have a clean commit history going forward. Generally, we want to avoid merge commits and do a rebase instead. I left some comments. Otherwise, @adamjonas can confirm he's satisfied with the modal
> 
> it's still says a merge confilcts?

No. I'm saying it's best if you don't have merge commits in your git history. We prefer a rebase for clean commit history

**Comment by 0tuedon** [comment]
Hey @Extheoisah  i would be closing this PR now because of the git history commit and would be creating a new one with the effects.

**Pull request closed without merging**